server/server.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-console */
+import express from 'express';
+
+const app = express();
+app.get('/', (req, res) => res.send('Hello World!'));
```

Comment:
I want you to abstract the second part of the argument of `app.get()`. And by abstraction, I mean to assign it to a variable outside the `app.get()` and then use that variable inside the `app.get`

Reply:
Okay

Reply:
Another thing i noticed after i ran npm start and my console message shows, the terminal kinda freezes like its still running some background process. i have to close the terminal or open another one. Is there a way to fix this?

Reply:
@knite51 That is perfectly normal (as you will soon find out). That simply means your server is now running, and you can make requests to the server with another client. I think I'd have to be on a call with you to explain some of these actually.